<WIP> Necromancer patch july

WrathOfGrapes opened the pull request:
- [x] Remove Crossbowmen and swordsmen.
- [x] Forlorn hope: minorstartaff 10->20
- [x] Wretches: -1 dmg, price 10 -> 25
- [x] Abominations: incorporate burst 3 -> burst 2.
- [x] Raise dead pool: Add three ranged templates. Not necessary to remove anything. There are 9 melee and 1 ranged already. Copystats with offset does not work, so will need to stat manually. Maybe 3 bows and 1 sling for variety.
- [x] Make Pierce the Veil of Death free to learn. Not worth a full ritual, but does serve a purpose for Ghost Mages. Also just a neat little flavor thing.
- [x] Major Invocation: cost 75 hands -> 60 hands. I think it can go as low as that with no problem. Overcasting it is already a trap.
- [x] Greater Shade: Range 6 on breath (is a bit weak)
- [x] Eyes of the Bodak range 8 (is a bit weak)
- [ ] Gulthias Tree and Passage to the Underworld: Agree they should functionally combine. Is an annoying flavor break to require Hades Gates to be cast in living ancient forests though. Just to preserve atmosphere: Gulthias Tree ritual unlocks Passage to the Underworld ritual for free, Passage ritual gets nostart. Flavor text slightly adjusted (eating the fruit of the gulthias tree reveals the path to the underworld or some such - I can write that). Cost 25 + 100. Also gives the option of annoying Druids without spending the full 125 hands.
- [ ] Mummification: move to T1 but make rarestart and afterprev Grim invocation. Fairly bad to start with that one.
- [ ] Set one of either wall of unlife or pyramid of skulls to always learn after Major Invocation (requires disabling original ritual and defining a new version with afterprev). To further increase chance of getting the summoning ritual early.
- [ ] Flavor text for Blights
- [ ] Move Planar Inversion from Lich to Demi Lich

WrathOfGrapes commented:
@johanstender Could you review and finish missing items?

WrathOfGrapes commented:
Need help with these. Have some issue with ritual modding

- [ ] Gulthias Tree and Passage to the Underworld: Agree they should functionally combine. Is an annoying flavor break to require Hades Gates to be cast in living ancient forests though. Just to preserve atmosphere: Gulthias Tree ritual unlocks Passage to the Underworld ritual for free, Passage ritual gets nostart. Flavor text slightly adjusted (eating the fruit of the gulthias tree reveals the path to the underworld or some such - I can write that). Cost 25 + 100. Also gives the option of annoying Druids without spending the full 125 hands.
- [ ] Set one of either wall of unlife or pyramid of skulls to always learn after Major Invocation (requires disabling original ritual and defining a new version with afterprev). To further increase chance of getting the summoning ritual early.
- [ ] Flavor text for Blights
- [ ] Move Planar Inversion from Lich to Demi Lich

WrathOfGrapes commented:
> Minor technical hickup.
> 
> 1. I tried editing your recruitment changes directly to add catapults, captains etc, but got a "file cannot be edited". When I try  "edit file" again, it looks like my changes are saved though. I am navigating this wrongly I think.
> 2. When merging the full pull request it It tells me "head branch is out of date". Maybe bc I did something on the recruitment commit? I don´t know.

Branch history looks weird but the changes seem to be there
https://github.com/Conquest-of-Elysium-Mods/elysium-remastered/pull/21/files#diff-08e94313627c268641b2f6cce129a983aaf76ac95c388a74c5529a48b691a530R2285

johanstender commented:
Haha, not its fine! I just can´t keep up. The time constraints I mentioned are a pretty real thing :-) 

- Ziz used to be front and had an annoying tendency to just flap into defenders and die. So became mid to at least preserve it as a siege weapon. Can be reversed if you prefer, but try it out first.  
- Villains + wretches: agree
- raise legion on graveyards: Yeah, mechanically I can see it, flavor-wise not sure. Obvious result would be a lot of soulless/longdead, which seems to be exactly what animate dead already does.  

I´ll do the rituals, probably tonight, so you can see how they work.

johanstender commented:
ok so just a workflow question: Now branch is merged to main. Best way to proceed would be 1) Delete and make new branch for the ritual edits, or  2) keep working in this one and merge again once done?

WrathOfGrapes commented:
The second one. And I think it's better to merge it into the main when all the changes are finalised and implemented. No reason to rush it and split into two patches. We have a very small amount of changes left to finish the patch.

WrathOfGrapes commented:
> - Ziz used to be front and had an annoying tendency to just flap into defenders and die. So became mid to at least preserve it as a siege weapon. Can be reversed if you prefer, but try it out first.  

I think it's generally a bad practice to give mid rank to melee-only units. "Front-" should do the job so it would stay behind troops but still engage when there is noone left (or does it fly over the frontline?)


> - raise legion on graveyards: Yeah, mechanically I can see it, flavor-wise not sure. Obvious result would be a lot of soulless/longdead, which seems to be exactly what animate dead already does.  
I think you can avoid generic soulless and longdead while still not making this ability too much. You're sacrificing a graveyard, you deserve something good.


> I´ll do the rituals, probably tonight, so you can see how they work.
I've spent a lot of time looking at other rituals' code in .c5m files. I feel like I'm doing everything right but it still doesn't work. Don't forget to play test it :)


I think
